test/DependenciesBlockVariable.unittest.js

```diff
@@ -1,92 +0,0 @@
-"use strict";
```

Comment:
any reason why we are removing this test?

Reply:
This unit test is too specific. It tests the implementation internals instead of the overall logic. Moreover integration tests are more relevant in this case I think.